.golangci.yml

```diff
@@ -58,18 +56,21 @@ linters:
     - gocyclo
     - dupl
 linters-settings:
+  gofumpt:
+    lang-version: "1.17"
+    extra-rules: true
   revive:
     ignore-generated-header: true
     severity: warning
   gomnd:
     ignored-numbers: # Why we have a big range of file permissions
-      - '0644'
-      - '0655'
-      - '0666'
-      - '0770'
-      - '0755'
-      - '0765'
-      - '0777'
+      - '0o644'
```

Comment:
check on the impact of the converting to octal notation & doesn't seem to be intuitive for a user too

Reply:
This is the issue from the go where gofmt talks about making the octal notion clear: https://github.com/golang/go/issues/33363

Reply:
Golang version after 1.13 should prefer the octal notation to make it clear.
const perm = 0o755